docker-compose.yml

```diff
@@ -66,7 +66,9 @@ services:
       - ./assets:/var/www/blogchain/assets
 
   truffle:
-    build: bc
+    build:
+      context: .
+      dockerfile: bc/Dockerfile
```

Comment:
This is here only because it was broken before and I didn't rebuild it until now.

Reply:
Actually, I may move it to a separate PR.